app/src/main/res/layout/fragment_bottom_sheet.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The name of this file should match its location since all layout resources are global to the app.